puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/foreman-discovery-image-publish.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -xe
+
+# monkey patch vagrant-rackspace - bug https://github.com/mitchellh/vagrant-rackspace/pull/153
+sed -i 's/retryable(:on => Fog::Errors::TimeoutError, :tries => 200) do/retryable(:on => [Excon::Error::Timeout, Fog::Errors::TimeoutError], :tries => 200) do/' /home/jenkins/.vagrant.d/gems/gems/vagrant-rackspace-0*/lib/vagrant-rackspace/action/create_server.rb
+
+base_dir=/var/www/vhosts/downloads/htdocs/discovery
+
+cd aux/vagrant-build
+
+export distro=f27
+export proxy_repo=$(eval echo ${proxy_repository})
+export VAGRANT_DEFAULT_PROVIDER=rackspace
```

Comment:
I wasn't aware that this existed. This provider no longer exists on slaves and the monkey patch above also no longer works.

Reply:
Can they safely be removed without consequence?